static/js/util.js

```diff
@@ -254,6 +254,47 @@ exports.prefix_sort = function (query, objs, get_item) {
              rest:    noMatch };
 };
 
+exports.emoji_prefix_sort = function (query, objs, get_item) {
```

Comment:
There is a lot of repeated logic in here. Is there any way to call prefix sort above, and then just modify the result?

Reply:
Sure I'll look into it :+1: